messages/en-US.json

```diff
@@ -28,7 +28,11 @@
   "pricing.interest-free": "interest-free",
   "loading": "Loading…",
   "shipping.label": "Calculate shipping",
-  "shipping.eta": "up to {eta} days",
+  "shipping.eta": "up to {eta} {especification}", 
```

Comment:
shouldn't it be `specification` instead of `especification`?

Reply:
heheheh sim